assets/src/legacy/attributeTable.js

```diff
@@ -1511,17 +1510,31 @@ var lizAttributeTable = function() {
                         }
                     } else {
                         // Check if we need to replace url or media by link
+                        let davConf = lizUrls['webDavUrl'] && config.layers[aName]?.webDavFields && Array.isArray(config.layers[aName].webDavFields) && config.layers[aName].webDavFields.includes(columnName) || false;
                         colConf['render'] = function (data, type, row, meta) {
                             // Replace media and URL with links
                             if (!data || !(typeof data === 'string'))
                                 return data;
+                            if (davConf) {
+                                // replace the root of the url
+                                if(data.startsWith(lizUrls['webDavUrl'])){
+                                    data = data.replace(lizUrls['webDavUrl'],'dav/')
```

Comment:
I'm not a PHP dev, but is-it possible to have a static variable for `dav/` ?
It's used in many places : L1521, L1532, lizmap/modules/lizmap/lib/Request/WMSRequest.php lizmap/modules/view/classes/popup.class.php, lizmap/modules/view/controllers/media.classic.php ? (or where it's appropriate if any)

Because this is a kind of "overpass" specific in Lizmap.
It's look easier to debug in IDE : right click, search usages in case of a refactoring etc
Thanks if it's possible, otherwise, skip :)

@mdouchin Please confirm if possible.

Reply:
@Gustry
**js**: I've not a strong opinion on this. If `'dav/'` is used only in `attributeTable.js ` for now I would leave it in plain text, but if it could be used in other contexts could you point me out the right place to eventually set this variable (`lizMap` object or `lizMap.config` object maybe? ). In this case, also the `media/` string could be stored in a variable?

**php**: you can have an utility class that expose public static properties, in this case I would use `RemoteStorageRequest` class directly

Reply:
Oh yes, one was in JS indeed.
I saw you did in PHP, thanks. I find it easier to debug later, to have something centralized, or helpful in IDE.

Yes, I think `media/` might like 10 years old in Lizmap... At that time, the source code was way smaller, so that's why it wasn't done, always a good idea to start new good practice ;-)

Reply:
@Gustry,
ok, I'm going to add a property in `lizUrls`, seems to me that could be the right place to store stuff about... urls.
I'm not going to add a property for `/media` in this PR because I would keep things distinct

Thanks

Reply:
Of course, I didn't ask about `media`, it was just to explain the context, 10 years ago.